client/client-app/models/EventModel.js

```diff
@@ -4,8 +4,33 @@ class EventModel {
         this.name = name;
         this.date = date;
         this.time = time;
-        this.imgSrc = imgSrc;
+        this.imgSrc = imgSrc; // When we begin to add images, we should probably move 
+                              // image representation away from a file path. Maybe a database ID?
         this.description = description;
     }
+
+    get id() { return this._id; }
```

Comment:
Is there a reason we're using getters and setters? It feels a bit Java-y

Reply:
Nah your right, we can probably just access the individual components. I can make a commit to main later.